### PR TITLE
Add lua/r/health.lua

### LINKED ISSUE #251: Not inside R or Python code chunk.
For .Rmd files, a similar behaviour as described in https://github.com/R-nvim/R.nvim/issues/190 occurs. 
I.e. a new .Rmd file with 
```{r}
a <- 1 *cursor*
```
gives "Not inside R or Python code chunk".
Temporarily, :InspectTree fixes this, but this does not work for long. 

Modifying line 577 of r/send.lua to 

if lang ~= "r" and lang ~= "markdown" then

seems to fix this.

### PR DESCRIPTION
**Pull request by jalvesaq**
Close #251

### CODE REVIEW

**Comment by jalvesaq** [comment]
The health module is called by Neovim even if not editing an R file. This means that we can't check the validity of user options such as  `R_app`. I didn't add a check for `tmux` because it's possible to run R in an external `kitty` terminal emulator without `tmux`. Should you check if `tmux` is installed anyway? Should we add any other test to the module?

File: lua/r/health.lua [hunk 1]
```diff
@@ -0,0 +1,69 @@
+--- Adapted from markview.nvim plugin
+
+local M = {}
+
+local ts_available, treesitter_parsers = pcall(require, "nvim-treesitter.parsers")
```
Comment (PMassicotte) [inline lua/r/health.lua:5]: Should it be moved inside the check() function?

**Comment by PMassicotte** [comment]
Very nice idea. we should have had this long time ago, good jog.

> Should you check if `tmux` is installed anyway?

Maybe just a warning?

> Should we add any other test to the module?

Good question. I can not think of anything else at the moment.

**Comment by PMassicotte** [comment]
Looks good to me for merging now.

**Comment by jalvesaq** [comment]
Thanks! I merged it, maybe too quickly... I was also thinking that Tmux could be added as a warning, but we can add more diagnostics in the future.

**Pull request merged**